cli/tauri.js/bin/tauri.js

```diff
@@ -22,53 +33,66 @@ const tauri = function (command) {
     if (process.argv && !process.env.test) {
       process.argv.splice(0, 3)
     }
-    runOnRustCli(command, process.argv || []).promise.then(() => {
+    runOnRustCli(
+      command,
+      (process.argv || []).filter((v) => v !== '--no-update-notifier')
+    ).promise.then(() => {
       if (command === 'init') {
         const {
           installDependencies
         } = require('../dist/api/dependency-manager')
         return installDependencies()
       }
     })
-    return
-  }
+  } else {
+    if (
+      !command ||
+      command === '-h' ||
+      command === '--help' ||
+      command === 'help'
+    ) {
+      console.log(
+        chalk.green(`
```

Comment:
but green isn't anywhere in our logo:
<img width="422" alt="image" src="https://user-images.githubusercontent.com/35242872/113008999-daa03400-9177-11eb-9ffd-727cd99b6031.png">

closest there is cyan i guess

Reply:
my bad, for some reason, I have tauri associated with green in my head XD

Reply:
![image](https://user-images.githubusercontent.com/48618675/113009798-919caf80-9178-11eb-9553-f6f50633b1d4.png)
or 
![image](https://user-images.githubusercontent.com/48618675/113009867-a24d2580-9178-11eb-8bfa-173797268145.png)

Reply:
Nice. I guess I like the second one better (blue logo, yellow details)

Reply:
alright, I will push it